chat-widget/automation_tests/e2e/areas/livechat/lcwWebchatwithSpecialSymbols.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { TestSettings } from "configuration/test-settings";
+import { LiveChatPage } from "pages/livechat";
+import { testHelper } from "pages/testHelper";
+import { BrowserContext } from "playwright";
+import { SelectorConstants } from "Utility/constants";
+
+describe("LCW webchat with special symbols", () => {
+    let liveChatContext: BrowserContext;
+    let livechatpage: LiveChatPage;
+
+    beforeEach(async () => {
+        liveChatContext = await browser.newContext({
+            viewport: TestSettings.Viewport,
+            acceptDownloads: true,
+        });
+    });
+
+    afterEach(async () => {
+        testHelper.dispose(liveChatContext);
+    });
+
+    ///<summary>
+    /// Test Case 2698882: [LCW Customization] [WebChat] verify able to use special symbols asterisk(*) and the pound(#)
+    /// Test case link: https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/2698901
```

Comment:
lets not use any Microsoft resource reference anywhere in git gub repo.  lets just use TC Id and title for tracking